src/bgw/job.c

```diff
@@ -390,28 +390,28 @@ ts_bgw_job_get_scheduled(size_t alloc_size, MemoryContext mctx)
 			continue;
 		}
 #endif
-
 		/* handle NULL columns */
-		value = slot_getattr(ti->slot, Anum_bgw_job_hypertable_id, &isnull);
```

Comment:
Reordered the initialization order to match the order of the `FormData_bgw_job` struct.